include/moveit_visual_tools/moveit_visual_tools.h

```diff
@@ -347,6 +347,16 @@ class MoveItVisualTools : public rviz_visual_tools::RvizVisualTools
                               const std::string& name,
                               const rviz_visual_tools::colors& color = rviz_visual_tools::GREEN);
 
+  /**
+   * \brief Create a MoveIt collision rectangular cuboid at the given pose
+   * \param pose - position of the centroid of the cube
+   * \param x - width
+   * \param y - depth
+   * \param z - height
```

Comment:
the standard in this file is to call these variables width, depth, height - not x,y,z which imply position

Reply:
done